src/commands/azure-node.browse-packages.js

```diff
@@ -0,0 +1,60 @@
+var fs = require('fs');
+var vscode = require('vscode');
+var npmUserPackages = require('npm-user-packages');
+var jsonEditor = require('../codegen/jsoneditor');
+var utils = require('../utils');
+var codegen = require('../codegen/codegen.browse-packages');
+
+exports.createCommand = function createCommand() {
+  vscode.commands.registerCommand('Azure-Node.browse-packages', function () {
+    npmUserPackages('windowsazure').then(data => {
```

Comment:
ToDo: Move this to extension load from command invocation. Two advantages:
1. perf will be better if we didn't make a webrequest after command invocation.
2. prefetching and caching will also ensure we don't make a request on every command invocation but once per VSCode session.